src/packages/spinner/ducks.ts

```diff
@@ -23,22 +22,14 @@ type SpinnerState = {
 const initialState: SpinnerState = {};
 
 export default createReducer(initialState)
-  .handleAction(startSpinnerAction, (state, { payload: id }) =>
-    id === NO_SPINNER
```

Comment:
Why removing `NO_SPINNER`? How do we make an api call without spinner now?

Reply:
takto https://github.com/Kamahl19/react-starter/pull/205/commits/551ad8da838e23de0138e312fdf77c86ec2b422b?file-filters%5B%5D=.ts#diff-77ec4935a866cae9b425b6ad79ba5d15R37

Reply:
Spinner funguje 3 sposobmi:
1. ak definujes apiCallId, zobrazi sa ti konkretny spinner
2. ak nedefinujes apiCallId, zobrazi sa ti Global spinner
3. ak posles NO_SPINNER, nezobrazis globalny spinner

Spinner je naschval opt-out a nie opt-in, lebo sa v kazdom projekte zabudalo na spinneri. `NO_SPINNER` je treba vratit alebo navrhni lepsie riesenie

Reply:
3. je ako 1. naco by som posielal no_spinner, to uz poslem rovno specificky nazov toho callu, len nevyuzijem selector, cize bude to no spinner.

2. nemame poriesenu. ak to tak ma byt, potom som spravil zle lebo dispatchnem akciu len ak je apicallid defined. Cize len setnem global za default apicallid